s3.go

```diff
@@ -42,13 +43,14 @@ func NewS3(ctx context.Context, config config.Config, disk string) (*S3, error)
 	region := config.GetString(fmt.Sprintf("filesystems.disks.%s.region", disk))
 	bucket := config.GetString(fmt.Sprintf("filesystems.disks.%s.bucket", disk))
 	url := config.GetString(fmt.Sprintf("filesystems.disks.%s.url", disk))
+	token := config.GetString(fmt.Sprintf("filesystems.disks.%s.token", disk), "")
```

Comment:
The token is not fixed data, right? In current logic, the token will be fixed when initiating S3, I wonder if it's expected.

Reply:
most of the cases accesskeyId and accessKeySecret will be used. But on lambda those two does not work and lambda provide a temporary access token for the time span of lambda function. So it will be injected via environmet > goravel config > s3 driver. 

As lambda instances are short term so it will be destroyed and recreated with new token after few minutes so no worry here. 

Let me know if you mean something else.

Reply:
So the token is fixed in the short term; it's safe to inject it via environment directly.